### PR TITLE
Feature/nex 662/newsql storage implementation

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
https://oat-sa.atlassian.net/browse/NEX-662
Spike:
https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/174788466/Back-port+Item+revision+versioning+in+BackOffice+

for testing on VM or kubernetes
please add this to your setup.json

    "taoRevision": {
      "storage": {
        "type": "configurableService",
        "class": "\\oat\\taoRevision\\model\\storage\\NewSqlStorage",
        "options": {
          "persistence": "default"
        }
      }
    }

### CODE REVIEW

File: model/storage/RdsStorage.php [hunk 1]
```diff
@@ -209,13 +216,13 @@ public function getRevisionsDataByQuery($query)
 
     /**
      * @param array $statement
-     * @param string $modelid
+     * @param string $modelId
```
Comment (boajer) [inline model/storage/RdsStorage.php]: Should it be string or array?

**Comment by boajer** [comment]
Any possiblity to add tests?

**Comment by mike-ionut-mihai-sandu-tao** [comment]
the tests are not present because of the lack of time,  i will do them as soon as possible when i am able to get a little bit of time :)

File: config/default/repository.conf.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (siwane) [inline config/default/repository.conf.php]: This file is not required as it is already registered, see `manifest > install > php`
You can choose this solution (default config files) or install by script.

File: config/default/storage.conf.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
```
Comment (siwane) [inline config/default/storage.conf.php]: This file is not required as it is already registered, see `manifest > install > php`
You can choose this solution (default config files) or install by script.

**Pull request merged**